ping/rust/Cargo.toml

```diff
@@ -7,13 +7,11 @@ version = "0.1.0"
 async-std = {version = "1.10", features = ["attributes", "tokio1"]}
 env_logger = "0.9.0"
 if-addrs = "0.7.0"
-ipnetwork = {version = "0.19.0", default-features = false, features = ["serde"]}
 log = "0.4"
 rand = "0.8"
-serde = {version = "1", features = ["derive"]}
 serde_json = "1"
 soketto = "0.7.1"
-testground = {git = "https://github.com/testground/sdk-rust", rev = "94a9a72796f94cc7ca786a5f019d07f328c76d4b", version = "0.4.0"}
+testground = {git = "https://github.com/testground/sdk-rust", branch = "master", version = "0.4.0"}
```

Comment:
Should we pin this?
If tomorrow, `rust-sdk` publish a new version, this will break the compilation everywhere.

Reply:
I released `v0.4.0` yesterday. I suggest we use the released version instead.

https://github.com/testground/sdk-rust/releases/tag/v0.4.0

Reply:
> I released `v0.4.0` yesterday. I suggest we use the released version instead.

Of course you are maintaining that too :joy: